script/prerender/fleet.ts

```diff
@@ -396,9 +396,9 @@ async function waitUntilWorkDone(
       '[SQS] [GetQueueAttributes] Unexpected response: missing Attributes key'
     );
 
-    const numMessages = parseInt(attributes.ApproximateNumberOfMessages, 10) +
-      parseInt(attributes.ApproximateNumberOfMessagesDelayed, 10) +
-      parseInt(attributes.ApproximateNumberOfMessagesNotVisible, 10);
```

Comment:
I'd say it's probably better to just use `assertDefined` for each of these 3 like the attributes key above, just in case...